services/retrieve_and_rank/v1.js

```diff
@@ -51,20 +56,20 @@ module.exports = function (RED) {
         var params = {
           training_data: msg.payload
         };
-      
+
         if (config.rankername) {
-          params.training_metadata = "{\"name\": \""+config.rankername+"\"}";
+          params.training_metadata = '{\"name\": \"'+config.rankername+'\"}';
         }
-        node.status({fill:"blue",shape:"ring",text:"Uploading training data"});
+        node.status({fill:'blue',shape:'ring',text:'Uploading training data'});
         retrieve_and_rank.createRanker(params, function(err, res) {
             handleWatsonCallback(null,node,msg,err,res,function() {
-              node.status({fill:"blue",shape:"ring",text:"Training data uploaded. Ranker is training"});
+              node.status({fill:'blue',shape:'ring',text:'Training data uploaded. Ranker is training'});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2690663401/issues/source?bid=3330694&fileBranchId=3336646#l66)